web/src/main/webapp/WEB-INF/pages/application.jsp

```diff
@@ -28,6 +28,7 @@
 
                     <label id="positionLabel" class="control-label text-left" for="drop">
                         <spring:message code="application.create.position.label"/>
+                        <span id="requiredMark">*</span>
```

Comment:
Do not use the same value for the `id` attribute for more than one element on a page!

Use a css `class` instead.